bot.py

```diff
@@ -17,37 +18,44 @@
 
 #add readyAt property to bot class
 def readyAtGetter(self):
-	return self._readyAt
+    return self._readyAt
 def readyAtSetter(self,value):
-	self._readyAt = value
+    self._readyAt = value
 commands.Bot.readyAt = property(readyAtGetter, readyAtSetter)
 @bot.event
 async def on_ready():
-	await bot.change_presence(status=discord.Status.online, activity=discord.Game(f'{bot.command_prefix}help for commands'))
-	bot.readyAt = datetime.utcnow()
-	print (f"Bot online")
-	
-'''
-#kind of not a fan of this
+    await bot.change_presence(status=discord.Status.online, activity=discord.Game(f'{bot.command_prefix}help for commands'))
+    bot.readyAt = datetime.utcnow()
+    print (f"Bot online")
+
+
+
 @bot.event
 async def on_command_error(ctx, error):
-	if isinstance(error,commands.errors.CommandNotFound):
-		return
-	await ctx.send("An error occured!\n```{}```".format(error))
-'''
-curdir = os.getcwd()
-curdir.split('\\')
-if "Synergile" not in curdir:
-	os.chdir("Synergile")
-else:
-	pass
+    if isinstance(error,commands.errors.CommandNotFound):
+        return
+    else:
+        channel = bot.get_channel(817127799256252437)
+        with open("error.log", "w") as err:
+            err.write(str(traceback.format_exception(type(error), error, error.__traceback__)))
```

Comment:
```suggestion
            err.write("".join(traceback.format_exception(type(error), error, error.__traceback__)))
```
Fixes janky writing to log file attachment